src/js/views/globals/search/patient-search_views.js

```diff
@@ -42,11 +42,34 @@ const PatientSearchPicklist = Picklist.extend({
   itemTemplate: hbs`
     {{matchText text search}}{{~ remove_whitespace ~}}
     <span class="patient-search__picklist-item-meta">{{formatDateTime birth_date "MM/DD/YYYY"}}</span>
+    {{#each identifiers}}<span class="patient-search__picklist-item-meta">{{this}}</span>{{/each}}
   `,
   itemTemplateContext() {
+    // REMOVE: for sandbox testing purposes only
+    // this.model.set(
+    //  'identifiers',
+    //  [{ 'type': 'PID', 'value': '0123456789' }, { 'type': 'SSN', 'value': '321-45-6789' }, { 'type': 'XID', 'value': '9876543210' }],
+    // );
+
+    const settings = this.state.get('settings');
```

Comment:
could maybe be `const settingsIdentifiers = get(settings, 'result_identifiers')`;  If it's just going into an underscore function it doesn't need the default `[]`.  Otherwise there's a logic branch that needs testing.  Although..  I'm really tempted to start using `settings?.result_identifiers`